Data/RAList.hs

```diff
@@ -230,6 +231,10 @@ import Prelude hiding(
     zip, zipWith, unzip
     )
 import qualified Data.List as List
+import Control.DeepSeq (NFData(rnf))
+
+-- import Control.DeepSeq (NFData, rnf)
+
 #if !MIN_VERSION_base(4,9,0) == 1
```

Comment:
hrmm, this is something we should look at, i dont remember writing CPP that looks like that :(